### ISSUE TITLE
Better ttc support

### ISSUE THREAD

**Issue by stenson** [goodhertz/coldtype#3]
like an index on `Style` that lets you specify a member of a ttc? right now seems to just use the... first? one

**Comment by HinTak** [comment]
Don't know to what extent you depend on skia-python for this, but see https://github.com/kyamagu/skia-python/issues/138 for the upstream problem, and recent addition of a workaround.

**Comment by stenson** [comment]
thanks for the note — we don't currently use skia for anything font related in coldtype. I tried a while ago to incorporate some skia typography facilities via skia-python but found it very buggy. All font-reading in coldtype is currently done with fontTools

**Comment by HinTak** [comment]
I would agree that skia is quite buggy in that department, but it is partly the underlying OS's problem. I recently discovered that Blink (which acts as a layer between Chrome and Skia) configure Skia to load fonts via FreeType on non-linux, instead of via the host's CoreText (mac) or directwrite (windows) when mac or Windows's own font loading is limiting. Hence the new API added for the ttc workaround, which is based on what blink does to skia when skia is used inside chrome.

**Comment by HinTak** [comment]
In general, I found skia to be faster when it works correctly for the purpose, than cairo.
